src/orion/core/cli/plot.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Module running the plot command
+========================
+
+Exposes the interface for plotting for command-line usage.
+
+"""
+import argparse
+import logging
+
+import orion.core.io.experiment_builder as experiment_builder
+from orion.core.cli import base as cli
+from orion.storage.base import get_storage
+import orion.plotting.base
+
+# c'est quoi que ça fait ça?
+#from orion.storage.base import get_storage
+
+
+log = logging.getLogger(__name__)
+DESCRIPTION = "Produce plots for Oríon experiments"
+
+
+
+def add_subparser(parser):
+    """Add the subparser that needs to be used for this command"""
+    plot_parser = parser.add_parser("plot", help=DESCRIPTION, description=DESCRIPTION)
+
+    cli.get_basic_args_group(plot_parser)
+
+    plot_parser.add_argument(
+        "kind",
+        type=str,
+        help="kind of plot to generate. "
```

Comment:
It would be good to move [this]( https://github.com/Epistimio/orion/blob/develop/tests/unittests/plotting/test_plot_accessor.py) inside orion/plotting/base.py and pass this list here to argument `choices`.

Reply:
Done.